src/openrct2/util/Util.h

```diff
@@ -44,7 +44,7 @@ char* safe_strcpy(char* destination, const char* source, size_t num);
 char* safe_strcat(char* destination, const char* source, size_t size);
 char* safe_strcat_path(char* destination, const char* source, size_t size);
 char* safe_strtrimleft(char* destination, const char* source, size_t size);
-#if !(defined(_GNU_SOURCE) || (defined(__DARWIN_C_LEVEL) && __DARWIN_C_LEVEL >= 200809L))
+#if !(defined(_GNU_SOURCE) || (defined(__DARWIN_C_LEVEL) && __DARWIN_C_LEVEL >= 200809L) || defined(__FreeBSD__))
```

Comment:
To be consistent `__DARWIN_C_LEVEL` abomination replace `__FreeBSD__` with `__BSD_VISIBLE`. That would make the conditional respect `-D_XOPEN_SOURCE=<...>` or `-D_POSIX_C_SOURCE=<...>`.